tutorials/llm/llama-3/pruning-distillation/04_a_distilling_depth_pruned_student.ipynb

```diff
@@ -0,0 +1,136 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "09d30e35-8e9d-4d2e-bd14-738c627a3963",
+   "metadata": {},
+   "source": [
+    "### Step 4: Distill knowledge from teacher into student\n",
+    "Distillation of a model with NeMo Framework is also possible using a python script: [megatron_gpt_distillation.py](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/megatron_gpt_distillation.py). In this notebook, we will explore distillation with the depth-pruned model as the `STUDENT` model. \n",
```

Comment:
fix capitalization

  "Distillation of a model with NeMo Framework is also possible using a Python script: [megatron_gpt_distillation.py](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/megatron_gpt_distillation.py). In this notebook, we will explore distillation with the depth-pruned model as the `STUDENT` model. \n",